### PR TITLE
Updated run.sh to pass in the mysql key & trust stores.

### PR DESCRIPTION
**Pull request by kshakir**
Also updated jenkins with file that appears in the dev /etc/cromwell.conf.

### CODE REVIEW

**Comment by kshakir** [comment]
Tex-- assuming liquibase has been successfully run in the past, this `run.sh`, combined with the latest jenkins config file, _should_ start up the server cleanly. If you could sanity check run for me, I'd appreciate it.

**Comment by mcovarr** [comment]
So... what exactly was not working before?

File: src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala [hunk 1]
```diff
@@ -38,6 +39,16 @@ object SlickDataAccess {
...
   implicit class ConfigWithUniqueSchema(val config: Config) extends AnyVal {
     /**
+     * Returns either the "url" or "properties.url"
+     */
+    def urlKey = if (config.hasPath("url")) "url" else "properties.url"
+
+    /**
+     * Returns the value of either the "url" or "properties.url"
+     */
+    def urlValue = config.getString(urlKey)
```
Comment (mcovarr) [inline src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala]: why would we expect one or the other?

File: src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala [hunk 2]
```diff
@@ -81,10 +92,13 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
...
 
   // NOTE: if you want to refactor database is inner-class type: this.dataAccess.driver.backend.DatabaseFactory
-  val database = Database.forConfig("", databaseConfig.withUniqueSchema)
+  private val configWithUniqueSchema = databaseConfig.withUniqueSchema
+  val database = Database.forConfig("", configWithUniqueSchema)
 
   // Possibly create the database
   {
+    import SlickDataAccess._
+    log.info(s"Running with database ${configWithUniqueSchema.urlKey} = ${configWithUniqueSchema.urlValue}")
```
Comment (mcovarr) [inline src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala]: I know @coreone had concerns about passwords being logged, not sure if that would happen here.

Comment (kshakir) [inline src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala]: While setting any property via the url is technically [supported](http://dev.mysql.com/doc/connector-j/en/connector-j-reference-configuration-properties.html#connector-j-reference-url-format), we should *never* be setting `user` and `password` via the `url` in production.

For now, to keep it simple, we should embed the `useSSL` and `requireSSL` in the `url`, and keep using `driver`.

For our Slick/HikariCP/JDBC/MySQL integration, eventually I'd love to refactor and from `driver` to `dataSourceClass` (see doc under [`Database.forConfig`](http://slick.typesafe.com/doc/3.0.0/api/index.html#slick.jdbc.JdbcBackend$DatabaseFactoryDef@forConfig(String,Config,Driver):Database)). At that time we'll also have to from `url` to using `properties.url`, `properties.useSSL` and `properties.requireSSL`. There are also probably a [bunch](https://github.com/brettwooldridge/HikariCP/wiki/MySQL-Configuration) of other values we should tune too.

File: src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala (hunk 1 above)
Comment (kshakir) [inline src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala]: When using HikariCP, the default Slick 3.0.0 connection pool, with the newer `javax.sql.DataSource`, the `urlKey` is  `properties.url`. For other combinations, the key is `url`.

> The following additional keys are supported for HikariCP only:
> • dataSourceClass (String, optional): ... Note that url is ignored when this key is set (You have to use properties to configure the database connection instead).
> …

-- [Database.forConfig](http://slick.typesafe.com/doc/3.0.0/api/index.html#slick.jdbc.JdbcBackend%24DatabaseFactoryDef@forConfig(String,Config,Driver):Database)

**Comment by kshakir** [comment]
Not 100% sure what wasn't working at what point.

I suspect that based on the order of the original commits<sup>1</sup>, the `RunMysql` and server should have both worked at "4.". At that point I believe the config `url` still contained `useSSL=true`, the application config was being passed on the command line, and the mysql jdbc code should have been in the main assembly.

By the time I was running "11." earlier today, the configuration `url` no longer contained `useSSL=true`, and connections within `SlickDataAccess` were returning the error combo:

```
java.sql.SQLTimeoutException: Timeout after 1000ms of waiting for a connection.
...
Caused by: java.sql.SQLException: Access denied for user '…'@'…' (using password: YES)
```

I did add another variable in "11." by always testing with `useSSL=true&requireSSL=true`, but according to the [logs](http://pastebin/209) of the latest 'RunMysql', `jdbcMain` and `jdbcRequireSsl` passed. So that _shouldn't_ have changed the results.

Meanwhile, all test combinations of setting ssl worked for both slick and raw datasource connections, in tests via the url (\*Ssl\*), or via the dataSource properties (\*Prop). So I think just setting back the `useSSL=true` is the minimum required fix, but I'd prefer to see `requiredSSL=true` added as well, as was successfully run in `slickSslDriver`.

<sup>1</sup> What I believe is the previous order of the commits:

1. Updated run.sh to pass in the mysql key & trust stores.
2. log database config
3. make mysql not test-only
4. Add config file option in run.sh to make container use custom configuration
5. debugging "script"
6. log actual uniquified config
7. Test at JDBC level.
8. hardcode use of SSL
9. count rows in WORKFLOW_EXECUTION
10. Logging the just the URL in SlickDataAccess, not the entire config.
11. Added a suite of mysql ssl test.

**Comment by kshakir** [comment]
Received a verbal tentative :+1: from @mcovarr (I will follow up with more inline docs regarding conversation above) and :+1: from @coreone with his latest `JAVA_OPTS` commit.

This jenkins build may break in develop due to server-conf-file logic required from #102, but we believe it _should_ all work once that's merged in too.

Merging.

**Pull request merged**